src/syntax-tree/ast.ts

```diff
@@ -2328,14 +2346,23 @@ export class Module {
 
         try {
             if (code instanceof TypedEmptyExpr) {
-                const parent = code.getParentStatement(); //line that contains "code"
-                if (parent.hasScope()) {
-                    refs.push(...parent.scope.getValidReferences(code.getSelection().startLineNumber));
-                } else {
-                    //code lines at the very top level will not have a body so use Module's scope
-                    refs.push(...this.scope.getValidReferences(code.getSelection().startLineNumber));
+                let scope = code.getParentStatement()?.scope; //line that contains "code"
```

Comment:
This check needed to be adjusted to actually catch statements that have a scope and not include ones without one.